specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2018-08-01-preview/costmanagement.json

```diff
@@ -2961,6 +2964,11 @@
             "modelAsString": true
           },
           "readOnly": true
+        },
+        "acknowledged": {
+          "description": "Determines whether the alert has been acknowledged.",
+          "type": "boolean",
+          "readOnly": true
```

Comment:
what is the value of status property when acknowledged property is true?

Reply:
These properties are independent - acknowledged could be true/false while the status property has any value.
For example:
- An alert is created with an 'active' status. Acknowledged is false by default. The user then acknowledges the alert, but it remains active as long as it's relevant.
- An alert could be overridden / resolved with or without it being acknowledged.